tests/DependencyInjection/Fixtures/config/xml/full.xml

```diff
@@ -67,6 +67,133 @@
+                                },
+                                {
+                                    "keyId": { "$binary": { "base64": "X78UZZ/HTX2wLw4K3uG42w==", "subType": "04" } },
+                                    "path": "collectionField",
+                                    "bsonType": "objectId"
+                                },
+                                {
+                                    "keyId": { "$binary": { "base64": "LugQL/ZXTJOl856Yacmkwg==", "subType": "04" } },
+                                    "path": "boolField",
+                                    "bsonType": "bool"
+                                }
+                            ]
+                        },
+                        "encrypted.patients": {
+                            "fields": [
+                                {
+                                    "keyId": { "$binary": { "base64": "GH25/XvYSaCgTUQLAo1hQw==", "subType": "04" } },
+                                    "path": "pathologies",
+                                    "bsonType": "array"
+                                },
+                                {
+                                    "keyId": { "$binary": { "base64": "krVWyFlNTUOaGFMfk+s7UA==", "subType": "04" } },
+                                    "path": "patientRecord.billing",
+                                    "bsonType": "object"
+                                },
+                                {
+                                    "keyId": { "$binary": { "base64": "X1ZaSI1GSAKnZ+sPGcmYBA==", "subType": "04" } },
+                                    "path": "patientRecord.billingAmount",
+                                    "bsonType": "int",
+                                    "queries": { "queryType": "range", "contention": 8, "min": 100, "max": 2000, "sparsity": 1, "trimFactor": 4 }
+                                }
+                            ]
+                        },
+                        "encrypted.client": {
+                            "fields": [
+                                {
+                                    "keyId": { "$binary": { "base64": "I0Aw18vnRGWzVS1t3uejpQ==", "subType": "04" } },
+                                    "path": "name",
+                                    "bsonType": "string"
+                                },
+                                {
+                                    "keyId": { "$binary": { "base64": "XSPRK3vaTLmMZr9IEj/qwQ==", "subType": "04" } },
+                                    "path": "clientCards",
+                                    "bsonType": "array"
+                                }
+                            ]
+                        }
+                    }
+                    ]]>
+                </doctrine:encryptedFieldsMap>
+                <doctrine:extraOptions>
+                    <doctrine:mongocryptdURI>mongodb://localhost:27020</doctrine:mongocryptdURI>
```

Comment:
I think there's a case for making all but `mongocryptdSpawnArgs` attributes. Nested elements should only be needed when multiple values are accepted.

See [`search-index-field`](https://github.com/doctrine/mongodb-odm/blob/2.11.x/doctrine-mongo-mapping.xsd) in the ODM's mapping XSD for prior art.